### DISCUSSION TITLE
Add an option to avoid loading aja plugin?

### DISCUSSION THREAD

**Discussion by sacereda** [obsproject/obs-studio#7527] [Development Questions] [answered]
Hi,

Is there any way to skip loading the aja module when launching obs? The device might be in use, and CardEntry::Initialize will unconfigure the card even in that case:

https://github.com/obsproject/obs-studio/blob/77b11c16f1bb6a38abbc8e0e18308aa0c6466e41/plugins/aja/aja-card-manager.cpp#L75

I'm not sure if there is any way to blacklist an specific module, or maybe using CNTV2DriverInterface::AcquireStreamForApplicationWithReference to actually check if the board is in use.

Thanks,
 Sergio

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds like a bug report, can you please open an issue instead?

**Comment by sacereda** [comment]
Sure, thanks!
